decomp2d/transpose_z_to_y.f90

```diff
@@ -33,7 +33,7 @@ subroutine transpose_z_to_y_real(src, dst, opt_decomp)
 #endif
     
     integer :: s1,s2,s3,d1,d2,d3
-    integer :: ierror
+    integer :: ierror, istat
```

Comment:
Isn't it simpler to use ierror instead of defining a new integer ? The situation is similar line 162.

Reply:
We could. The code from NVIDIA came with this flag and therefore for consistency i use it. Once i have everything fix. I can change that and add all the errors checks.